packages/adapter-sveltekit/src/hooks/project-init.ts

```diff
@@ -173,6 +173,8 @@ const createSliceSimulatorPage = async ({
 		return;
 	}
 
+	// Due to limitations in Svelte 4 on how slots can be typed, we cannot provide a Svelte 5-flavor
+	// version of this snippet, so we have to rely on the legacy Svelte 4 syntax.
```

Comment:
What is the limitation? Is there a page in the Svelte docs that explain it?

Reply:
Sorry, it's poorly explained in the comment.

Basically the @prismicio/svelte essentially being a Svelte 4 codebase, it uses slots to pass data to child components which cannot be typed in a compatible way in Svelte 5 because of the introduction of snippet. Making a major of @prismicio/svelte will solve this issue.